.github/workflows/bsim-tests-publish.yaml

```
@@ -25,4 +25,4 @@ jobs:
           commit: ${{ github.event.workflow_run.head_sha }}
           event_file: event/event.json
           event_name: ${{ github.event.workflow_run.event }}
-          files: "bsim-test-results/**/bsim_results.xml"
+          files: "results/bsim_{bluetooth,net}/bsim_results.xml"
```

I think you misunderstood https://github.com/zephyrproject-rtos/zephyr/pull/56944#discussion_r1171162620. I meant no change is necessary in this line (keep `bsim-test-results/**/bsim_results.xml`).

```suggestion
          files: "bsim-test-results/**/bsim_results.xml"
```